data_pump/collection.py

```diff
@@ -0,0 +1,109 @@
+import logging
+
+from utils import read_json, convert_response_to_json, do_api_post
+
+
+def import_collection(metadata, group_id, handle, community_id, collection_id,
+                      collection2logo, imported_handle, metadatavalue, metadata_field_id, statistics):
+    """
+    Import data into database.
+    Mapped tables: collection, community2collection, metadatavalue, handle
+    """
+    json_name_col = 'collection.json'
+    json_name_com2col = 'community2collection.json'
+    json_name_metadata = 'metadatavalue.json'
+    url_col = 'core/collections'
+    url_step = 'core/collections/'
+    importedColl = 0
+    importedGroup = 0
+    json_a = read_json(json_name_col)
+    comm_2_coll_json = read_json(json_name_com2col)
+    coll2comm = dict()
+    if not comm_2_coll_json:
+        logging.info("Community2collection JSON is empty.")
+        return
+    for i in comm_2_coll_json:
+        coll2comm[i['collection_id']] = i['community_id']
+
+    # because the role DEFAULT_READ is without old group id in collection
+    coll2group = dict()
+    metadata_json = read_json(json_name_metadata)
+
+    if metadata_json:
+        for i in metadata_json:
+            if i['resource_type_id'] == 6 and 'COLLECTION_' in i['text_value'] and '_DEFAULT_READ' in i[
+                    'text_value']:
+                text = i['text_value']
+                positions = [ind for ind, ch in enumerate(text) if ch == '_']
+                coll2group[int(text[positions[0] + 1: positions[1]])] = i['resource_id']
+    if not json_a:
```

Comment:
See the comment above, you can put this in front of it and remove the second condition.